XIVSlothCombo/CustomCombo.cs

```diff
@@ -499,9 +499,9 @@ protected static bool CanWeave(uint actionID, double weaveTime = 0.7)
         /// without causing clipping and checks if you're casting a spell to make it mage friendly
         /// </summary>
         /// <param name="actionID">Action ID to check.</param>
-        /// <param name="weaveTime">Time when weaving window is over. Defaults to 0.7.</param>
+        /// <param name="weaveTime">Time when weaving window is over. Defaults to 0.5.</param>
         /// <returns>True or false.</returns>
-        protected static bool CanSpellWeave(uint actionID, double weaveTime = 0.7)
+        protected static bool CanSpellWeave(uint actionID, double weaveTime = 0.5)
```

Comment:
Making a special exception for `CanSpellWeave` here. We will be leaving `CanWeave` at `0.7` for the foreseeable.